node/sc/vt_recovery_test.go

```diff
@@ -608,7 +608,7 @@ func requestKLAYTransfer(info *testInfo, bi *BridgeInfo) {
 
 	opts := bi.account.GetTransactOpts()
 	opts.Value = big.NewInt(testAmount)
-	_, err := bi.bridge.RequestKLAYTransfer(opts, info.aliceAuth.From)
+	_, err := bi.bridge.RequestKLAYTransfer(opts, info.aliceAuth.From, big.NewInt(0))
```

Comment:
How about implement unit tests for `fee > 0`?

Reply:
In `bridge_manager_test.go`, I added the TC with `fee > 0`.
I think in VTR, we don't need to add the TC. Because VT, VTR uses same handling logic. :)